src/event.js

```diff
@@ -554,32 +582,52 @@ function leverageNative( el, type, forceAdd ) {
 	jQuery.event.add( el, type, {
 		namespace: false,
 		handler: function( event ) {
-			var saved = dataPriv.get( this, type );
+			var maybeAsync, result,
+				saved = dataPriv.get( this, type );
 
 			// Interrupt processing of the outermost synthetic .trigger()ed event
-			if ( ( event.isTrigger & 1 ) && !saved ) {
+			if ( ( event.isTrigger & 1 ) && this[ type ] && !saved ) {
 
 				// Store arguments for use when handling the inner native event
-				dataPriv.set( this, type, slice.call( arguments ) );
+				saved = slice.call( arguments );
+				dataPriv.set( this, type, saved );
 
 				// Trigger the native event and capture its result
+				// Support: IE <=9 - 11+
+				// focus() and blur() are asynchronous
+				maybeAsync = allowAsync( this, type );
```

Comment:
The second argument to `allowAsync` doesn't seem to be used anywhere. Can we drop it?

Reply:
Yes, but that would increase the size. And since it also fits flexibility patterns from elsewhere in the library, I'd like to keep it.

Reply:
OK, fair enough.